src/main/plugin/iso19139.ca.HNAP/layout/config-editor.xml

```diff
@@ -502,6 +502,24 @@
                    or="distributionFormat"/>
           </section>
 
+          <action type="add" name="distributionFormat" or="distributionFormat"
+                  in="/gmd:MD_Metadata/gmd:distributionInfo/gmd:MD_Distribution">
+            <template>
+              <snippet>
+                <gmd:distributionFormat>
+                  <gmd:MD_Format>
+                    <gmd:name>
+                      <gco:CharacterString>DOC</gco:CharacterString>
```

Comment:
Sould we be defaultiong the name to DOC?  It may be better to leave this empty.

Reply:
yeah. i tried empty string. It's not giving me error but put empty to that field. It was defaulted as DOC when we new a metadata. But I think empty for second distribution format is also ok. I changed the code.